l10n-pocket/en/pocket/about.ftl

```diff
@@ -0,0 +1,52 @@
+
+# Variables:
+#   $pocket_home_attrs (string) attributes for a HTML link
+#   $pocket_home_domain (string) GetPocket.com - attrs for hyperlink
+#   $pocket_and_firefox_attrs (string) - attrs for hyperlink
+#   $pocket_daily_newsletter_attrs (string) - attrs for hyperlink
+#   $pocket_app_attrs (string) - attrs for hyperlink
+pocket-about-our-curators-vet-the-most-fascinating = Our curators vet the most fascinating finds to ensure they’re from a diverse range of publications with a track record of trustworthy and accurate coverage, and we share them on <a { $pocket_home_attrs }>{ $pocket_home_domain }</a>, <a {$pocket_and_firefox_attrs}>{ -brand-name-pocket } for { -brand-name-firefox }</a>, in our <a { $pocket_daily_newsletter_attrs }">daily newsletter</a>, and within the <a { $pocket_app_attrs }>{ -brand-name-pocket } app</a>. Our recommendations are available primarily in English and German, though we look forward to sharing in more languages in the future.
+
+pocket-about-use-pocket-your-way = Use { -brand-name-pocket } your way
+
+pocket-about-there-are-lots-of-ways = There are lots of ways you can use { -brand-name-pocket }. Some people use it to save stories throughout the day to read when they’re ready. Hunter-gatherer types use it to save all the things they want to read, watch, listen to, or buy. Others use { -brand-name-pocket } to highlight key passages to help them grow, learn, and thrive.
+
+pocket-about-some-folks-love-exploring = And some folks love exploring { -brand-name-pocket }’s recommendations to find inspiration, challenge their views, and see the occasional cat (or dog) story.
+
+pocket-about-know-its-good = Know it’s good
+pocket-about-internets-only-as-good-as = The internet’s only as good as the content you encounter, so — by saving and consuming with intent — together we make { -brand-name-pocket } a brighter corner of the internet. And, as part of the { -brand-name-mozilla } and { -brand-name-firefox } family, we do all this with the same dedication to privacy you’ve come to expect from { -brand-name-firefox } and { -brand-name-mozilla }.
+
+pocket-about-welcome-to-the-sunny-side = Welcome to the sunny side of the web.
+pocket-about-start-using-pocket = Start using Pocket
+
+# Variables:
+#   $press_email_link (email link) mailto:press@getpocket.com
+#   $press_email (string) press@getpocket.com
+pocket-about-contact-us-press = Contact us for press inquiries at <a href="{ $press_email_link }">{ $press_email }</a>.
+
+# Variables:
+#   $download_url (url) https://blog.getpocket.com/press/
+#   $presspage_id (string) ID of HTML element
+pocket-about-download-press-kit = Download our images and press kit <a id="{ $presspage_id }" href="{ $download_url }">here</a>.
```

Comment:
There's another ID in the L10n string here (up to you if you want to fix like the other string, just a nice to have).